### ISSUE TITLE
[example] A connected citation graph of open access papers

### ISSUE THREAD

**Issue by rht** [davidar/TeX.js#11]

**Comment by davidar** [comment]
@rht Uh, what?

**Comment by rht** [comment]
Apologize if unclear.
Explanation: ⓐ is a unicode symbol for open access (much like copyright or trademark). To unpack, this is an example request for an open access paper which citations are open access papers, which citations are open access papers, all the way down (even so, the issue title captures what I meant better).

Surely a cluster of oa papers can be found, though not common. For example a paper with one or two or no citations.

**Comment by davidar** [comment]
Yes, that would be cool. I don't think it's relevant to this project, but I would like to do something with citation graphs in the future, so will keep this in mind :)

**Comment by rht** [comment]
Hmm...Isn't it relevant for the case of rendering the citation links?
It would be better if the example is _an_ oa paper with oa citations.

What is out of scope, is distilling out all the connected oa papers from arxiv.

**Comment by davidar** [comment]
Ah, indeed. A little more description in the OP would have helped ;)
